src/bot/modules/webhooksender.js

```diff
@@ -53,8 +53,8 @@ module.exports = async pkg => {
       }
     }).catch(async e => {
       if (e && e.code && !(e.code == '50035' || e.code == '10015')) {
-        global.logger.warn(`Got ${e.code} while sending webhook to ${pkg.guildID} (${global.bot.guilds.get(pkg.guildID) ? global.bot.guilds.get(pkg.guildID).name : 'Could not find guild!'})`)
-        global.webhook.warn(`Got ${e.code} while sending webhook to ${pkg.guildID} (${global.bot.guilds.get(pkg.guildID) ? global.bot.guilds.get(pkg.guildID).name : 'Could not find guild!'})`)
+        global.logger.warn(`Got ${e.code} while sending webhook to ${pkg.guildID} (${guild ? guild.name : 'Could not found the guild!'})`)
```

Comment:
There is a bot.guilds.get check here because it will return the latest result of if the bot is in the server (updated in the background by the library). Guild here is just an instance of the Guild at hand and does not necessarily mean it still exists.